master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -1,142 +1,102 @@
+1. Deny all egress traffic
+1. Allow egress traffic to kube-dns
+1. Cleanup Namespace
 
-We'll use a new namespace for this guide.  Run the following command to create it.
+### 1. Create the Namespace and Nginx Service
 
-```
-kubectl create ns advanced-policy-demo
-```
-
-And then enable isolation on the namespace using a [default policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/#default-policies).
-
-```
-kubectl create -f - <<EOF
-kind: NetworkPolicy
-apiVersion: extensions/v1beta1
-metadata:
-  name: default-deny
-  namespace: advanced-policy-demo
-spec:
-  podSelector:
-    matchLabels: {}
-EOF
-```
-
-#### Run an nginx Service
-
-We'll run an nginx Service in the namespace.
+We'll use a new namespace for this guide.  Run the following commands to create it and a plain nginx service listening on port 80.
 
 ```shell
+kubectl create ns advanced-policy-demo
 kubectl run --namespace=advanced-policy-demo nginx --replicas=2 --image=nginx
 kubectl expose --namespace=advanced-policy-demo deployment nginx --port=80
 ```
 
-#### Check using calicoctl
-
-> **Note**: This requires the [calicoctl tool to be configured]({{site.baseurl}}/{{page.version}}/reference/calicoctl/setup/etcdv2).
-> For example: `export ETCD_ENDPOINTS=http://10.96.232.136:6666`.
-{: .alert .alert-info}
-
-Now that we've created a namespace and a set of pods, we should see those objects show up in the
-Calico API using `calicoctl`.
+#### Verify Access - Allowed All Ingress and Egress
 
-We can see that the namespace has a corresponding [profile]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/profile).
+Open up a second shell session which has `kubectl` connectivity to the Kubernetes cluster and create a busybox pod to test policy access.  This pod will be used throughout this tutorial to test policy access.
 
 ```shell
-$ calicoctl get profile -o wide
-NAME                          TAGS
-k8s_ns.advanced-policy-demo
-k8s_ns.default
-k8s_ns.kube-public
-k8s_ns.kube-system
-```
-
-Because all pods in the namespace are now selected, any traffic which is not explicitly allowed by a policy will be denied.
-
-We can see that this is the case by running another pod in the namespace and attempting to
-access the nginx Service.
-
-```
 $ kubectl run --namespace=advanced-policy-demo access --rm -ti --image busybox /bin/sh
+
 Waiting for pod advanced-policy-demo/access-472357175-y0m47 to be running, status is Pending, pod ready: false
 
 If you don't see a command prompt, try pressing enter.
-
-/ # wget -q --timeout=5 nginx -O -
-wget: download timed out
 / #
 ```
 
-We can also see that the two nginx pods are represented as [WorkloadEndpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint) in the Calico API.
+Now from within the busybox "access" pod execute the following commands to test access.
 
+```shell
+/ # wget -q --timeout=5 nginx -O -
+/ # wget -q --timeout=5 google.com -O -
```

Comment:
This command against `google.com` spits out a LOT of scary looking text. Can we find something that is more user-friendly?

Reply:
Perhaps not, in which case we'll just live with this for now.

Reply:
A simple `ping -c 1 google.com` looks nicer - what do you think about that?

```
/ # ping -c 1 google.com
PING google.com (173.194.74.100): 56 data bytes
64 bytes from 173.194.74.100: seq=0 ttl=52 time=0.938 ms

--- google.com ping statistics ---
1 packets transmitted, 1 packets received, 0% packet loss
round-trip min/avg/max = 0.938/0.938/0.938 ms

```

Reply:
My problem with ping is it uses an entirely different protocol (ICMP vs TCP) and is sometimes blocked by default in various environments (as is the case with acs-engine).  What if we switched up the google queries to something like this instead?  `wget --spider -S "google.com" 2>&1 | grep "HTTP/" | awk '{print $2}'`